app/src/main/java/com/tiptopgoodstudio/androidresources/ResourcesFragment.java

```diff
@@ -0,0 +1,100 @@
+package com.tiptopgoodstudio.androidresources;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class ResourcesFragment extends Fragment {
+
+    private static final String TAG = "Resources";
```

Comment:
Use the class name

Reply:
Got it, made change in the coming commit.